pepper-apis/dsm-server/src/resources/config/application.conf.default

```diff
@@ -0,0 +1,251 @@
+// 
+// The server's TimeZone
+//
+defaultTimezone = "UTC"
+
+//
+// Allowed cross-origin resource sharing hosts
+//
+corsOrigins = [
+  "http://localhost:4200"
+]
+
+//
+//
+//
+GoogleAnalytics.trackingId = "google-analytics-token"
+
+projectId =  "google-project-id"
```

Comment:
There are at least 4 Google Project ID keys:
* `projectId`
* `google.secret.project`
* `portal.googleProjectName`
* `pubsub.projectId`

In practice, all of them should be set to the same value.